cocotb/runner.py

```diff
@@ -186,10 +188,20 @@ def test(
         else:
             self.gui = bool(gui)
 
+        # When using pytest, use test name as result file name
+        pytest_current_test = os.environ.get('PYTEST_CURRENT_TEST', '')
```

Comment:
```suggestion
        pytest_current_test = os.environ.get("PYTEST_CURRENT_TEST", "")
```
`black` would do this.